src/api/goods.ts

```diff
@@ -10,10 +10,12 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
+      return goods.sort((a, b) => a.name.localeCompare(b.name)).slice(0, 5);
```

Comment:
it would be better to `slice` first and then add `sort`. that will help you to save the number of actions with data